react-graphql-apollo-tutorial.md

```diff
@@ -325,26 +323,28 @@ client
   .then(console.log);
 {{< /highlight >}}
 
-Once you start your application on the command line again, you should see the mutation result. The result should be encapsulated in a `addStar` object (the name of the mutation) which should reflect exactly the objects and fields that you have defined in the mutation: `starrable`, `id` and `viewerHasStarred`.
+The result should be encapsulated in a `addStar` object (the name of the mutation), which should reflect exactly the objects and fields that you have defined in the mutation: `starrable`, `id` and `viewerHasStarred`.
+
+You've completed another learning step by using only Apollo Client without any view-layer library. This is to avoid confusing the features of Apollo Client and React Apollo. 
 
-In the previous sections, you have done another learning step by using only Apollo Client without any view-layer library. That's hopefully the way how you won't confuse the features of Apollo Client and the connecting view-layer library React Apollo in the following sections. In conclusion, Apollo Client can be used as a standalone GraphQL client without connecting it to any view-layer such as React whatsoever. However, now it seems a bit dull to see the data only on the command line. That's why people invented HTML and CSS in the first place. So let's see how Apollo can be connected as your data-layer to your React view-layer in the following section.
+Remember, Apollo Client can be used as a standalone GraphQL client without connecting it to a view-layer like React, though it may seem a bit dull to see the data only on the command line. HTML and CSS can make the operation read for a user interface, so let's see how Apollo connects th data-layer to a React view-layer in the next section.
```

Comment:
"HTML and CSS can make the operation read for a user interface, so let's see how Apollo connects th data-layer to a React view-layer in the next section." to "That's why people invented HTML and CSS in the first place. So let's see how Apollo can be connected as your data-layer to your React view-layer in the following section."

Reply:
I think the HTML and CSS sentence is completely out of place here.  Anyone reading a book about advanced web development already knows why HTML and CSS were invented.  Sidenote: adding "in the first place" to the end of a sentence is always unnecessary.  Try reading the same sentence without the phrase, and you'll see it says the exact same thing.  As a writer, always eliminate words and phrases that don't add anything to the topic (i.e. furthermore, in addition, in the first place)

Reply:
Good tip!

You are correct. Then let's leave out HTML and CSS completely.